File: 2745-construct-the-longest-new-string/2745-construct-the-longest-new-string.swift
class Solution {
    func longestString(_ x: Int, _ y: Int, _ z: Int) -> Int {
        var ans = 0
        if x == y {
            ans = 2 * min(x, y)
        } else {
            ans = 2 * min(x, y) + 1
        }

        ans += z

        return ans * 2
    }
}

File: 2745-construct-the-longest-new-string/2745-construct-the-longest-new-string.php
class Solution {

    /**
     * @param Integer $x
     * @param Integer $y
     * @param Integer $z
     * @return Integer
     */
    function longestString($x, $y, $z) {
        $ans = 0;
        if ($x == $y) {
            $ans = 2 * min($x, $y);
        } else {
            $ans = 2 * min($x, $y) + 1;
        }

        $ans += $z;

        return $ans * 2;
    }
}
